lib/cloud_controller/metrics/prometheus_updater.rb

```diff
@@ -62,70 +78,46 @@ def update_thread_info(thread_info)
       update_gauge_metric(:cc_thread_info_event_machine_resultqueue_num_waiting, thread_info[:event_machine][:resultqueue][:num_waiting], 'EventMachine requests waiting in queue')
     end
 
-    def update_failed_job_count(failed_jobs_by_queue, total)
+    def update_failed_job_count(failed_jobs_by_queue)
       failed_jobs_by_queue.each do |key, value|
-        metric_key = :"cc_failed_job_count_#{key.to_s.underscore}"
-        update_gauge_metric(metric_key, value, "Failed jobs for worker #{key}")
+        update_gauge_metric(:cc_failed_jobs_total, value, "Failed jobs for worker #{key}", labels: { queue: key.to_s.underscore })
```

Comment:
Using labels for the different queues enables us to display each queue and also show the total number of failed jobs in the monitoring system, using a single metric. (https://prometheus.io/docs/practices/instrumentation/#use-labels)